Source/WebCore/editing/cocoa/FontAttributesCocoa.mm

```diff
@@ -37,40 +38,37 @@
 
 namespace WebCore {
 
-static NSString *cocoaTextListMarkerName(ListStyleType::Type style, bool ordered)
+static NSString *cocoaTextListMarkerName(ListStyleType::Type styleType, const String& styleIdentifier, bool ordered)
```

Comment:
Can we just pass the whole `ListStyleType` struct instead of 2 arguments?

Reply:
I've start doing that but then we have to add serialization for the struct, which will be used only for Cocoa case. With this approach we reduce code with the cost of only an extra function argument.

Reply:
Adding serialization seems fine to me?